### PR TITLE
Models associations

### PR DESCRIPTION
**Pull request by kkmanuu**
✔️ I have successfully created model classes (User, Post, Comment, and Like) based on the ERD diagram.
✔️ I have established associations to accurately represent data relationships.
✔️ I utilized the "rails c" command to populate the database with initial data.
✔️ I created at least one user and four posts, along with comments and likes.
✔️ I demonstrated how to perform standard CRUD operations for users, posts, comments, and likes.

### CODE REVIEW

File: app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :user
+  belongs_to :post
```
Comment (OLIPLICHE) [inline app/models/comment.rb]: - [x]  Hello, in looking carefully at the associations between models from the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png), I see that there is a repetition in your code which might affect your set up. I think you should use the code below.

belongs_to :author, class_name: 'User'
belongs_to :post

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @kkmanuu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Created and sets up associations between models 👍 
- The associations between models are set up correctly 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/comment.rb (hunk 1 above)
Comment (kkmanuu) [reply app/models/comment.rb]: @OLIPLICHE Done making the changes

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @kkmanuu , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Requested Change implemented 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**